plugins/handler/plugin_impl_handler.go

```diff
@@ -71,11 +71,14 @@ func (p *Plugin) AfterInit() error {
 	p.Log.Info("AfterInit")
 
 	ticker := time.NewTicker(objectstore.ObjectStoreReadyInterval)
+	timeout := time.After(objectstore.ObjectStoreReadyTimeout)
```

Comment:
Same with this one, though this was also done and merged as part of #136 I believe.